src/viewer.js

```diff
@@ -434,7 +434,7 @@ export class Viewer {
    * @param {boolean} awaitResponse
    * @return {(!Promise<*>|undefined)}
    * @package
-   * @export
+   * _export
```

Comment:
We do manual exports practically for everything else. So, we could do the same here.